src/features/services/queries.ts

```diff
@@ -85,11 +79,11 @@ export function useCreateToken() {
           [inputs.address]: amount,
         },
       };
-      network?.tokens.create(param);
+      return await network?.tokens.create(param);
```

Comment:
We weren't actually returning the response, so errors weren't being surfaced correctly. This is bug 1.